.github/workflows/cypress.yml

```diff
@@ -0,0 +1,41 @@
+name: Cypress Tests
+on: 
+  workflow_run:
+    workflows: [Unit Tests, Linter]
+    types:
+      - completed
+    branches:
+      - 'main'
+jobs:
+  Chrome:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run on Staging 
+        uses: cypress-io/github-action@v2
+        with:
+          working-directory: client
+          browser: chrome
+          headless: true
+        env: 
+          NPM_CONFIG_PRODUCTION: false
+          CYPRESS_TEST_USER_EMAIL: ${{ secrets.CYPRESS_TEST_USER_EMAIL }}
+          CYPRESS_TEST_USER_PASS: ${{ secrets.CYPRESS_TEST_USER_PASS }}
+          CYPRESS_baseUrl: "https://tn-spa-bootstrapper-staging.herokuapp.com/"
+  Firefox:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run on Staging 
+        uses: cypress-io/github-action@v2
+        with:
+          working-directory: client
+          browser: firefox
+          headless: true
+        env: 
+          NPM_CONFIG_PRODUCTION: false
+          CYPRESS_TEST_USER_EMAIL: ${{ secrets.CYPRESS_TEST_USER_EMAIL }}
+          CYPRESS_TEST_USER_PASS: ${{ secrets.CYPRESS_TEST_USER_PASS }}
+          CYPRESS_baseUrl: "https://tn-spa-bootstrapper-staging.herokuapp.com/"
```

Comment:
Upon this merging to staging, it'll run the Cypress tests against staging. For that to work, Github Actions needs the creds that are here and Staging needs to have this user exist (which it does). For convenience these creds can be found in the Heroku settings, which is useful should you want to point your local Cypress at staging to test something.

Reply:
There are a couple of challenges with this.
Right now Heroku doesn't auto deploy because of the Github linkage bug.
But even if it did, Cypress is likely to run before the deploy finishes.
In the past I've add an arbitrary 5min wait for that, but I didn't want to pollute the bootstrapper with that.
I have some ideas for a better fix.